ttnn/cpp/ttnn/operations/ccl/reduce_scatter/device/host/reduce_scatter_full_worker_grid.cpp

```diff
@@ -376,7 +381,7 @@ static void add_worker_config_to_edm_builders(
         }
 
         // Get the maximum message size we'd like to use. Not the actual packet size
-        uint32_t expected_message_size_bytes = tensor_slicer.get_worker_slice_size_bytes(global_worker_idx);
+        uint32_t expected_message_size_bytes = clockwise_edm_builders.at(link).get_eth_buffer_size_bytes();//tensor_slicer.get_worker_slice_size_bytes(global_worker_idx);
```

Comment:
Except for this line, this PR looks good. The small packet optimization should be kept in place.